pkg/v1/guestcluster/guestcluster.go

```diff
@@ -83,7 +83,9 @@ func (s *Service) newKubernetesRestConfig(ctx context.Context, clusterID, apiDom
 	s.logger.LogCtx(ctx, "level", "debug", "message", "looking for certificate to connect to the guest cluster")
 
 	operatorCerts, err := s.certsSearcher.SearchClusterOperator(clusterID)
-	if err != nil {
+	if certs.IsTimeout(err) {
```

Comment:
Which timeout is configured? The default is 90 seconds IIRC but this is too long here IMO. 3 should be good to keep the reconciliation quick. WDYT?

Reply:
Agree it should be quick. I used 5 secs when testing but 3 secs is better.

Problem is the resource isn't configured yet. I don't want to do that until the chart resource that installs chart-operator is also ready.

Reply:
Ok cool.

Reply:
I'll add the configuration but hold off enabling the resource until the chart resource is ready.

Reply:
Scratch that I think it's better to do this when the resource is enabled. Going with this.